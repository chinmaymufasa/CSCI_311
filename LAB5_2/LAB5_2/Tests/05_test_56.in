2
175
5
2
33
5
2
65
6
2
122
6
9
3
65
6
9
2
162
6
9
3
620
6
9
-1
