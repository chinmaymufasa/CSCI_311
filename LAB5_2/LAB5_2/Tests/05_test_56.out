175 
33 175 
33 175 65 
33 122 175 65 
4
33 175 122 
3
33 162 175 122 
4
33 162 175 122 
4
